### DISCUSSION TITLE
Create risk analysis on Red Hat 8 Monarc installation

### DISCUSSION THREAD

**Discussion by grassel82** [monarc-project/MonarcAppFO#344] [Q&A] [answered]
Hi,

we have finally successfully installed Monarc on a Red Hat 8 Linux server.  We tried now to create a risk analysis and the fiels for selecting "Risk Model" stays empty.

We had a look at apache logs, but no hint on anything that goes wrong.

Could you point us in the wright direction to have a look? Are they loaded live from some Internet website as our server has no Internet access configured?

Kind regards,

Gilles

**Comment by ruslanbaidan** [comment, marked as answer]
Hello,

Normally you should see 2 default models (see screenshot), which are loaded from the "monarc_common" database. The database suppose to be installed and populated on your server during the installation process.
If you don't see them, please take a look at the Network tab and find a "models" request in your list, as drawn on the screenshot.
If there the response is empty or there is an error, you will be able to see it.

Kind regards

![Screenshot 2021-04-15 at 10 54 45](https://user-images.githubusercontent.com/3246171/114843240-e88dc000-9dd9-11eb-8903-29979733666c.png)

**Reply by grassel82** [reply]
Hi,

thank you for your reply. You were correct, no data in the response for the models request. I checked database content and there was an error when importing the data with the script "monarc_data.sql".

In my lab environment, I had a local MySQL installation, so there the import over CLI was successful, but in production environment we have a centralized MariaDB cluster and it is only available over the workbench application. execute SQL statements reports an error because he needs an DB assigned. I selected now the monarc_common DB to execute the script on.

But I saw that monarc_cli has an similard database structure. Is it necessary to run the monarc_data.sql statements alors on this DB?

Your help is greatly appreciated, thank you in advanced.

Kind regards,

Gilles

**Reply by ruslanbaidan** [reply]
You are welcome,

If you follow the installation steps and configure your db connections correctly:
https://github.com/monarc-project/MonarcAppFO/blob/master/INSTALL/INSTALL.debian10.md#database-connection
then running update_all.sh script should have done it:
https://github.com/monarc-project/MonarcAppFO/blob/master/INSTALL/INSTALL.debian10.md#3-update-monarc

Yes, it's mandatory to run all the migration and load the initial data, you can rerun "update_all.sh or do it manually.

**Reply by grassel82** [reply]
Everything ok now,

DB connections were ok, but SQL script execution did not work the first time in MySQL Workbench. You need to specify the correct DB because it will not work if not.  

I did rerun the update_all.sh script, everything ok now. Monarc is working now on Red Hat 8. So this is great news :-)

Kind regards,

Gilles

**Reply by ruslanbaidan** [reply]
Perfect, thank you for the information and best wishes.
